core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -2351,31 +2353,32 @@ private void freeAndJournal(RpcContext rpcContext, LockedInodePath inodePath, Fr
 
     try (InodeLockList lockList = mInodeTree.lockDescendants(inodePath, InodeTree.LockMode.WRITE)) {
       freeInodes.addAll(lockList.getInodes());
-      TempInodePathForDescendant tempInodePath = new TempInodePathForDescendant(inodePath);
-      // We go through each inode.
-      for (int i = freeInodes.size() - 1; i >= 0; i--) {
-        Inode<?> freeInode = freeInodes.get(i);
-
-        if (freeInode.isFile()) {
-          if (freeInode.getPersistenceState() != PersistenceState.PERSISTED) {
-            throw new UnexpectedAlluxioException(ExceptionMessage.CANNOT_FREE_NON_PERSISTED_FILE
-                .getMessage(mInodeTree.getPath(freeInode)));
-          }
-          if (freeInode.isPinned()) {
-            if (!options.isForced()) {
-              throw new UnexpectedAlluxioException(ExceptionMessage.CANNOT_FREE_PINNED_FILE
+      try (TempInodePathForDescendant tempInodePath = new TempInodePathForDescendant(inodePath)) {
```

Comment:
Is this just an indenting change?

Reply:
I added a try  resource block around tempInodePath creation, hence the indenting change.